extensions/roulette/roll/cog.py

```diff
@@ -99,6 +100,15 @@ def _is_admin(self, user: User | Member) -> bool:
         self.logger.debug(f"User {user.name}'s admin status: {is_admin}")
         return is_admin
 
+    async def _apply_timeout_roles(self, target, duration_label):
+        timeout_ids = config.timeout_roles()
+
+        for timeout_id in timeout_ids:
+            role = target.guild.get_role(int(timeout_id))
+
+            if role:
+                await target.add_roles(role, reason=f"Timed out for {duration_label} via Roulette")
```

Comment:
nit: Let's be semantic and indicate it's more like "adding role for duration timeout" -the timeout itself may occur in multiple (overlapping) fashions.